Exercises/3-unique.js

```diff
@@ -3,6 +3,14 @@
 // Create and return a new array without duplicate elements
 // Don't modify initial array
 
-const unique = array => [];
+const unique = array => {
+  const na = [];
+
+  array.forEach(i => {
```

Comment:
Use for..of here, it's much faster and suitable in this case.

Reply:
I thought I need to use Array functions mostly, so decided to use forEach